modules/acquia_cms_headless/src/Plugin/AcquiaCmsTour/AcquiaHeadlessForm.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Drupal\acquia_cms_headless\Plugin\AcquiaCmsTour;
+
+use Drupal\acquia_cms_tour\Form\AcquiaCMSDashboardBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+
+/**
+ * Plugin implementation of the acquia_cms_tour.
+ *
+ * @AcquiaCmsTour(
+ *   id = "acquia_cms_headless",
```

Comment:
Note for @nagwani / @vishalkhode1 Unless this ID matches our module, name, the wizard wouldn't properly pick it up.  The Dashboard had no issues.  Also the wizard is programed to use the module title rather our separate "Headless" title.

Reply:
@modularlab Ok. For now, let's use this.
@chandan-singh7929 I think we would need to fix this. As this shouldn't match with any drupal module name, it could be any unique name. Let me know your thoughts on this.